evennia/utils/evtable.py

```diff
@@ -574,9 +575,9 @@ def _align(self, data):
         hfill_char = self.hfill_char
         width = self.width
         if align == "l":
-            return [(line.lstrip() if self.justify else line) + hfill_char * (width - m_len(line)) for line in data]
+            return [(line.lstrip() if line.startswith(" ") and not line.startswith("  ") else line) + hfill_char * (width - m_len(line)) for line in data]
```

Comment:
... but on the other hand reverting the order of the if like this: 

```python
line if line.startswith("  ") else line.lstrip()
```
appears about twice faster than both.